Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -135,30 +188,111 @@ void HiresTexture::Update()
+  double percent_mem_to_cull = 30.0;  // aka culled_percent
+
+  if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) != 30.0)
+  {
+    if ((Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) < 5.0) ||
+        (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) > 90.0))
+    {
+      if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) < 5.0)
+      {
+        percent_mem_to_cull = 5.0;
+        OSD::AddMessage(
+            "Prefetcher: User INI setting for 'culled cercent' of " +
+                StringFromFormat(
+                    "%.1f %%", Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT)) +
+                " is below the lowest bound. Resetting to lowest allowable value.",
+            20000);
+        WARN_LOG(VIDEO, "Prefetcher: Mem limit cull percent INI override out of "
+                        "lower bound. Setting to lowest");
+      }
+      else if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) > 90.0)
+      {
+        percent_mem_to_cull = 90.0;
+        OSD::AddMessage(
+            "Prefetcher: User INI setting for 'culled percent' of " +
+                StringFromFormat(
+                    "%.1f %%", Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT)) +
+                " is below the highest bound. Resetting to highest allowable value.",
+            20000);
+        WARN_LOG(VIDEO, "Prefetcher: Mem limit cull percent INI override out of "
+                        "higher bound. Setting to highest");
+      }
+    }
+    else
+    {
+      // Successful Manual Override - Dont report values in OSD, routine stats notification will do
+      OSD::AddMessage("Prefetcher: Maximum delegated memory limit has been manually overriden "
+                      "by user INI settings.");
+      percent_mem_to_cull = Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT);
+      INFO_LOG(VIDEO, "Prefetcher: Mem limit overriden by user INI successfully, "
+                      "at %.2f (cull percent)",
+               percent_mem_to_cull);
+    }
+  }
+
+  double prefetched_size_sum = 0;  // total mem size incl. non-native decompressed textures
+  const double absolute_max_mem = Common::MemPhysical();
+  // Currently "total usable", excl. HW reserved
+  const double culled_percent_amount = (Common::MemPhysical() * percent_mem_to_cull) / 100;
+  const double delegated_max_mem = Common::MemPhysical() - culled_percent_amount;
+  const double below_min_req_mem = 3221225472;  // 3 GB; 2GB Rec. + 1GB OS/Misc
+  // WIP - using 1024 * ... etc but makes the line so long, no global ref found, make one?
+
+  // message only
+  if (absolute_max_mem <= below_min_req_mem)
+  {
+    OSD::AddMessage("Prefetcher: Warning! System RAM of " +
+                        StringFromFormat("%.2f MB", absolute_max_mem / (1024.0 * 1024.0)) +
+                        " is below Dolphin's minimum recommended specifications.",
```

Comment:
WIP, I don't what context that 2GB on Dolphin's FAQ is using. 2GB total installed, 2GB of free RAM (which means `installed - used = atleast 2GB free left`)